docs/node/workers.md

```diff
@@ -55,12 +61,79 @@ A Worker is in one of 7 states at any given point:
 
 If you need even more visibility into internal worker state, [see the API reference for more](https://nodejs.temporal.io/api/classes/worker.Worker).
 
-## Worker Security and Networking
+### Worker Security and Networking
 
 The Node SDK usually handles all of the communication between the Worker and the Temporal Server behind the scenes - no port configuration is required for development usecases.
 
+### Encryption at Rest
+
 In production settings, [Temporal supports mTLS encryption](/docs/server/security), required by Temporal Cloud.
 To configure this, this SDK exposes [the Rust Core SDK](https://github.com/temporalio/sdk-core) as `Core`, which you can configure before you run `workflow.create`:
 
 <!--SNIPSTART nodejs-mtls-worker-->
 <!--SNIPEND-->
+
+### Encryption in Transit
+
+You can use the advanced [Interceptor and Data Converter](/docs/node/interceptors) features to encrypt your inbound and outbound Workflow and Activity calls.
+
+## What is a Task Queue?
+
+import SharedTaskQueuesBasic from '../shared/task-queues-basic.md'
+
+<SharedTaskQueuesBasic
+workflowLink="/docs/java/workflows"
+workerLink="/docs/java/workers"
+/>
+
+### Where Task Queues are used
+
+In Node, a Task Queue is represented in code by name, as a `string`.
+There are 3 places where the name of the Task Queue is supplied by the developer.
+
+<details>
+<summary>
+
+1. When starting a Workflow, you must pass the `taskQueue` option to the [WorkflowClient's `createWorkflowHandle()` method](https://nodejs.temporal.io/api/classes/client.workflowclient#newworkflowhandle).
+
+</summary>
+
+```ts
+const workflow = workflowClient.createWorkflowHandle(myWorkflow, {
+  taskQueue: 'my-task-queue',
+});
+
+const result = await workflow.execute();
+```
+
+</details>
+<details>
+<summary>
+
+2. When creating a Worker, you must pass the `taskQueue` option to the [`Worker.create()` function](https://nodejs.temporal.io/api/classes/worker.worker-1#create).
+
+</summary>
+
+```ts
+const worker = await Worker.create({
+  workflowsPath: require.resolve('./workflows'),
+  taskQueue: 'my-task-queue',
+});
+```
+
+</details>
+<details>
+<summary>
+
+3. Optionally, in Workflow code, when calling an Activity, you can specify the task queue by passing the `taskQueue` option to [`createActivityHandle()`](https://nodejs.temporal.io/api/namespaces/workflow#configureactivities). If you do not specify a `taskQueue`, then the Node SDK places Activity Tasks in the same Task Queue as the Workflow Task Queue.
```

Comment:
There's also `createChildWorkflowHandle` that takes an optional taskQueue option.

Reply:
```suggestion
3. Optionally, in Workflow code, when calling an Activity, you can specify the task queue by passing the `taskQueue` option to [`createActivityHandle()`](https://nodejs.temporal.io/api/namespaces/workflow/#createactivityhandle) or [`createChildWorkflowHandle()`](https://nodejs.temporal.io/api/namespaces/workflow/#createchildworkflowhandle).
If you do not specify a `taskQueue`, then the Node SDK places Activity Tasks in the same Task Queue as the Workflow Task Queue.
```